pkg/conversion/converter.go

```diff
@@ -670,6 +670,32 @@ func (c *Converter) defaultConvert(sv, dv reflect.Value, scope *scope) error {
 		switch st.Kind() {
 		case reflect.Ptr, reflect.Interface:
 			return c.convert(sv.Elem(), dv.Elem(), scope)
+
+		case reflect.Slice:
```

Comment:
This is the right code for right now in this right place to fix one specific issue.

I'd rather see a design proposal on the next step in a post 1.2 world with a lot more time to discuss tradeoffs.